trillian.yaml

```diff
@@ -47,22 +47,23 @@ test:
       packages:
         - bash
         - go
+        - git
         - lsof
         - mysql
         - procps
         - ${{package.name}}-logserver
         - ${{package.name}}-logsigner
         - wget
+        - wait-for-it
   pipeline:
-    - uses: git-checkout
-      with:
-        repository: https://github.com/google/trillian
-        tag: v${{package.version}}
-        expected-commit: 205cc426984ed7bdc90cb242947cb94dcf3f945b
     - runs: |
+        git clone --depth 1 --branch v${{package.version}} https://github.com/google/trillian
```

Comment:
Is this so we can skip the expected-commit check?

Reply:
Yep, expected-commit is not needed since we're just cloning it for the test.